cleanup_32bit.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+from lxml import etree as ET
+import sys
+import ToolBase
+from osclib.core import fileinfo_ext_all
+from osclib.conf import Config
+from osclib.stagingapi import StagingAPI
+from osclib.cleanup_rings import CleanupRings
+
+class Cleanup32bit(ToolBase.ToolBase):
+    def run(self, prj: str, arch: str):
+        Config(self.apiurl, prj)
+        cr = CleanupRings(StagingAPI(self.apiurl, prj))
+        cr.whitelist = set(["wine", "wine-nine-standalone", "wine:staging"])
```

Comment:
Needed or requiredlist perhaps? whitelist is pretty generic ( and non inclusive) terminology

Reply:
It's actually a list in CleanupRings still, can be changed in a later refactor commit after I got this one working

Reply:
It's a dict now with more info, renamed to `force_required`